### PR TITLE
Fixes: [Bug] Multiple checkbox has same name #1

### LINKED ISSUE #1: [Bug] Multiple checkbox has same name
https://github.com/hiteshchoudhary/devui/assets/99522246/325a229a-8d41-4474-954e-a322ed5296ad

### PR DESCRIPTION
**Pull request by musiur**
Fixes #1 

Resolved:
Checkbox has unique name for ```htmlFor``` attributes with label to check one checkbox clicking on one label.

https://github.com/hiteshchoudhary/devui/assets/99522246/3dd38307-ec60-43dd-8a3e-eb1239b30179

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "@uiw/codemirror-themes-all": "^4.21.11",
     "@uiw/react-codemirror": "^4.21.11",
     "autoprefixer": "10.4.15",
+    "axios": "^1.5.0",
```
Comment (wajeshubham) [inline package.json]: Why did we install axios? I don't think there's a need for this package.

File: src/components/templates/authentication/register.jsx [hunk 2]
```diff
@@ -59,16 +59,16 @@ const Register = () => {
             <div className="relative mt-4 flex items-center">
               <div className="flex h-6 items-center">
                 <input
-                  id="notifications"
-                  aria-describedby="notifications-description"
-                  name="notifications"
+                  id="agree-to-receive-email"
```
Comment (wajeshubham) [inline src/components/templates/authentication/register.jsx]: No need to add such long ids, we can go with a common convention. For example, `checkbox-1`, `checkbox-2` and so on. Same for the `name` and `aria-describedby`

**Review by wajeshubham — COMMENTED** [review]
Kindly check the review comments

File: src/components/templates/authentication/register.jsx (hunk 2 above)
Comment (musiur) [reply src/components/templates/authentication/register.jsx]: I have changed the ID and names

image

File: package.json (hunk 1 above)
Comment (musiur) [reply package.json]: I have removed this.

**Comment by musiur** [comment]
I have resolved two of those issue of review.

- Removed axios
- Renamed ID and names of checkboxes

Fixes #1

**Review by wajeshubham — COMMENTED** [review]
Kindly resolve the merge conflicts

**Comment by musiur** [comment]
Fixes: #1 #2

Resolved the conflicts in` register.jsx` component. 

@wajeshubham

**Pull request merged**